Gemfile

```diff
@@ -8,13 +8,14 @@ gem 'sqlite3', '~> 1.4'
 # Use Puma as the app server
 gem 'puma', '~> 5.0'
 # Use SCSS for stylesheets
-gem 'sass-rails', '>= 6'
-# Transpile app-like JavaScript. Read more: https://github.com/rails/webpacker
-gem 'webpacker', '~> 5.0'
+gem 'sassc-rails'
+gem 'bootstrap'
```

Comment:
If you add bootstrap, you should probably add a version requirement so that you don't automatically get  new major version that you aren't expecting.

Reply:
ok, pinned to version 5